app/controllers/v1/inventory_access_requests_controller.rb

```diff
@@ -8,7 +8,9 @@ def index
 
   def create
     permission = Inventories::Permission.new(inventory: inventory, user: current_user)
-    @request = permission.request_access(role: create_request_params[:role])
+    # Note:  UI passes a list but the UI will never send back more than one element.
+    # UI has not been changed to preserve compatibility with existing code which assumes the same.
+    @request = permission.request_access(role: create_request_params[:roles].try(:[], 0))
```

Comment:
Isn't `try` cool? :) I mean, I'm sure Sandi Metz would be all like "omg nothing is something and this is a hidden conditional!!!111shift", and she does have a point, but it's so damn convenient.